IdentityCore/src/MSIDConstants.m

```diff
@@ -30,6 +30,10 @@
 NSString *const MSID_DEVICE_MODEL_KEY           = @"x-client-DM";
 NSString *const MSID_APP_NAME_KEY               = @"x-app-name";
 NSString *const MSID_APP_VER_KEY                = @"x-app-ver";
+NSString *const MSID_BROKER_RESUME_DICTIONARY_KEY   =   @"adal-broker-resume-dictionary";
+NSString *const MSID_BROKER_SYMMETRIC_KEY_TAG   = @"com.microsoft.adBrokerKey";
+NSString *const MSID_BROKER_V2_SCHEME = @"msauth";
+NSString *const MSID_BROKER_V3_SCHEME = @"msauthv2";
```

Comment:
Just wondering, why they are v3 & v2? Which scheme was v1?

Reply:
Current ADAL-broker protocol is 2. The one before it was 1 and used a bit different encryption logic. The new one is 3. Let me think if we can rename those more intuitively.

Reply:
maybe we should change scheme to msauthv3 then?

Reply:
Maybe. But that conflicts with the distinction of ADAL==v1 endpoint, MSAL==v2 endpoint. Broker protocol version is internal thing, vs this scheme has to be registered by the app to be able to query for it....

Reply:
Let me maybe just rename it to MSID_BROKER_ADAL_SCHEME and MSID_BROKER_MSAL_SCHEME?

Reply:
sounds go to me